.github/workflows/publish_image.yml

```
@@ -108,6 +115,7 @@ jobs:
           tags: ${{ steps.meta.outputs.tags }}
           no-cache: true
 
+    timeout-minutes: 15
```

actionlint (1.7.12)

[error] key "timeout-minutes" is duplicated in "release_arm" job. previously defined at line:78,col:5

(syntax-check)